Farid Masjedi/week02/HomeWork01/html-labs/busyhands/busyhandsCss.css

```diff
@@ -0,0 +1,160 @@
+body{
+  text-align: center;
+  margin: 0 auto;
+  background-color: black;
+  font-family:helvetica;
+}
+
+.wrapper{
+  background-color: white;
+  margin:0 auto;
+  max-width:960px;
+}
+.subwrapper{
+  max-width:800px;
+  margin:0 80px;
+}
+nav{
+  background-color: black;
+  padding:20px;
+  text-align:left;
+}
+nav a{
+  color:white;
+  text-decoration: none;
+  padding:10px;
+  margin-left:20px;
+}
+nav a:hover{
+  background-color: linear-gradient(90deg, rgba(0,0,255,0.1), rgba(0,0,255,0.8));
+  background: -webkit-linear-gradient(90deg, rgba(0,0,255,0.1), rgba(0,0,255,0.8));
+  background: -moz-linear-gradient(90deg, rgba(0,0,255,0.1), rgba(0,0,255,0.8));
+  text-decoration:overline;
+  padding-top:12px;
+}
+.logo,.social{
+  float:left;
+}
+#header-img::after{
+  content: '';
+  clear: both;
+  display: table;
+}
+.social{
+  position:relative;
+  left:300px;
+}
+.social:hover{
+  background:blue;
+  /* Question : How to make each one a button */
```

Comment:
Think more along the lines of how to style it to make it appear to be a button if it isn't actually a button. Adding things like a hover event help this.